app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+from typing import Dict
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def check_entry_requirements(self, visitor: Dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        vaccine = visitor["vaccine"]
+        expiration_date = vaccine.get("expiration_date")
+
+        if expiration_date < datetime.date.today():
```

Comment:
There is a potential issue here if 'expiration_date' is None. Consider adding a check to ensure 'expiration_date' is not None before comparing it with 'datetime.date.today()'.